packages/block-library/src/image/image.js

```diff
@@ -540,6 +540,13 @@ export default function Image( {
 		);
 	}
 
+	function onCaptionBlur() {
+		if ( captionFocused ) {
+			captionRef.current.blur();
```

Comment:
I don't think we need this ref.blur if https://github.com/WordPress/gutenberg/pull/30587 lands.